examples/integration/widgets/data_table_customization.py

```diff
@@ -3,22 +3,31 @@
                           NumberFormatter, StringFormatter, TableColumn)
 from bokeh.sampledata.periodic_table import elements
 
+
+def get_text_color(hex_color):
+    """Get a text color with high contrast v.s. the background color."""
+    r, g, b = int(hex_color[1:3], 16), int(hex_color[3:5], 16), int(hex_color[5:7], 16)
+    luminance = (0.299 * r + 0.587 * g + 0.114 * b) / 255
+    return '#000000' if luminance > 0.5 else '#ffffff'
```

Comment:
There is already `bokeh.colors.color.RGB.brightness()`, which uses a variation of this method. bokehjs has one slightly different as well. We need a well-defined and consistent API for this.

Reply:
I still think that luminance is a better metric for this, so I added it as `bokeh.colors.color.RGB.luminance`, and added the bokehJS equivalent too.

Now looks cleaner:

```python
'#000000' if RGB.from_hex_string(hex_color).luminance > 0.4 else '#ffffff'
```